docs/index.md

```diff
@@ -43,6 +43,78 @@ Once the installation has been successful, let's try running ```minimesos --help
 This should print the list of all possible commands and command line arguments.
 
 
+These are the options you might want to change to configure your cluster.
```

Comment:
Not sure why it was moved from here.
This was meant to be under the main configuration options